views/style.css

```diff
@@ -1,8 +1,8 @@
 slides.layout-widescreen > slide {
   width: 95vw;
-  height: calc(95vw / (1100 / 700));
+  height: 100%;
   margin-left: calc(95vw / -2);
-  margin-top: calc(95vw / (1100 / 700) / -2);
+  margin: -400px;
```

Comment:
This replaces a dynamically calculated value with a hardcoded one. Not sure that's universally better. Can you maybe instead improve the formula?

Reply:
That's right. I actually tried to eliminate hardcoded ratio. And with the last commit I shaped it a bit better. So instead of margin calculation i kept the border (margin) fixed to the screen and extract this gap from width and height of the slide `(calc(100% - margin))`. Please check the new commit changes.